cheroot/ssl/builtin.py

```diff
@@ -196,12 +420,24 @@ def env_dn_dict(self, env_prefix, cert_value):
         if not cert_value:
             return {}
 
-        env = {}
+        dn = []
+        dn_attrs = {}
         for rdn in cert_value:
             for attr_name, val in rdn:
                 attr_code = self.CERT_KEY_TO_LDAP_CODE.get(attr_name)
+                dn.append('%s=%s' % (attr_code or attr_name, val))
                 if attr_code:
-                    env['%s_%s' % (env_prefix, attr_code)] = val
+                    dn_attrs.setdefault(attr_code, [])
+                    dn_attrs[attr_code].append(val)
+
+        env = {
+            env_prefix: ','.join(dn),
+        }
+        for attr_code, values in dn_attrs.items():
+            env['%s_%s' % (env_prefix, attr_code)] = ','.join(values)
+            if len(values) > 1:
```

Comment:
Plz use "if not...: continue" style in order to reduce the following block indentation

Reply:
changed